spec/requests/cart/empty_cart_spec.rb

```diff
@@ -0,0 +1,52 @@
+require "rails_helper"
+
+RSpec.describe "Cart", type: :request do
+  describe "empty" do
+    let(:user) { create(:user) }
+    let(:book) { create(:book) }
+    let(:book2) { create(:book) }
+
+    def login(email, password)
+      post login_path, params: {
+        user: {
+          email: email,
+          password: password,
+        }
+      }
+    end
+
+    def add_item_to_cart(book_id)
+      post cart_path, params: {
+        cart: {
+          book_id: book_id
+        }
+      }
+    end
```

Comment:
We seem to have these methods in multiple tests, could they be extract to a shared helper?